src/main/groovy/life/qbic/datamodel/accounting/CostItem.groovy

```diff
@@ -0,0 +1,43 @@
+package life.qbic.datamodel.accounting
+
+/**
+ * An item with associated costs and quantity
+ *
+ * Cost items describe services and their price. They are the building blocks of invoices and offers.
+ * The unit price is always provided in euros.
+ * @since: 1.9.0
+ * @author: Tobias Koch
+ *
+ * * */
+class CostItem {
+
+    double quantity
+    final double UNIT_PRICE
```

Comment:
NIT: PACKAGE_PRICE

Reply:
or is it the price for the item?

Reply:
As far as I understand it the price is calculated from the packages, therefore I would not set it with the constructor but have a method to calculate it? But then we have no POJO, right?

Reply:
the unit price is the price per unit. so if the quantity is 2 then the total price for this item would be 2*unit_price

Reply:
unit = package? probably. But as I mentioned the term package is very QBiC specific and not well defined.

Reply:
ahhh this is the entity then... :D

Reply:
It is one line here http://www.h2officesolutions.com/h2os_custguide/image997.jpg